src/Symfony/Component/HttpFoundation/Request.php

```diff
@@ -89,10 +89,19 @@ class Request
     /**
      * Request body parameters ($_POST).
      *
+     * @deprecated since Symfony 6.2
+     *
      * @var InputBag
      */
     public $request;
 
+    /**
+     * Request body parameters ($_POST).
+     *
+     * @var InputBag
+     */
+    public $form;
```

Comment:
What about naming it `$postData`?

Reply:
the word `data` is implicit via the $request object no?
a request (class) transferring a data via http (foundation)
to me $post is good candidate :)

Reply:
That data is typically in the request body...so `$postBodyData`? 😆

Note that the same mechanism could work for other request methods, not just `POST`..